game/confusion.cc

```diff
@@ -0,0 +1,24 @@
+#include "confusion.h"
+#include "statInfo.h"
+#include "mobile.h"
+
+using namespace std;
+
+Confusion::Confusion(Hackmon &h) : Mobility(h), rng{RandomGenerator(1,10)}, 
+    damage{hackmon.stats.getMaxHP() * 0.1}, duration{5} {}
+
+
+bool Confusion::canMoveOverride() {
+  int currentHP = hackmon.stats.getStat(HP);
+  // 70% chance damage will be inflicted
```

Comment:
excuse me SIR. the comment says 70% chance but the code is actually 60% chance smh

Reply:
WHOOPS 🤷‍♂